src/engraving/rw/write/twrite.cpp

```diff
@@ -1616,10 +1616,12 @@ void TWrite::write(const Instrument* item, XmlWriter& xml, WriteContext&, const
         ClefTypeList ct = item->clefType(i);
         if (ct._concertClef == ct._transposingClef) {
             if (ct._concertClef != ClefType::G) {
-                if (i) {
-                    xml.tag("clef", { { "staff", i + 1 } }, TConv::toXml(ct._concertClef));
-                } else {
-                    xml.tag("clef", TConv::toXml(ct._concertClef));
+                if (ct._concertClef != ClefType::JIANPU && item->trackName() != "Jianpu") {
```

Comment:
Another doubtful string comparison... we should not rely on the value of `trackName()` of an instrument. This may be translated, or (in the future) user-edited, so shouldn't be used for logic. I also wonder why this condition appears _here_, and nowhere else.